haystack/document_stores/deepsetcloud.py

```diff
@@ -441,16 +445,44 @@ def write_documents(
         """
         raise NotImplementedError("DeepsetCloudDocumentStore currently does not support writing documents.")
 
+    def get_evaluation_sets(self) -> List[dict]:
+        """
+        Returns a list of uploaded evaluation sets to deepset cloud.
+
+        :return: list of evaluation sets as dicts
+                 These contain ("name", "evaluation_set_id", "created_at", "matched_labels", "total_labels") as fields.
+        """
+        return self.evaluation_set_client.get_evaluation_sets()
+
     def get_all_labels(
         self,
-        index: Optional[str] = None,
+        label_index: Optional[str] = None,
```

Comment:
I think this needs to be `index` to match the abstractmethod's signature. Besides that, I think we should accept `label_index` as an init-param too and take that if no index is being passed to these methods. This is actually how all the other DocumentStores work.

Reply:
I've switch it to use the `index` parameter from the __init__ function, see line 68.